docs/configuration/source/index.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
> Since we haven't extracted Kubernetes as plugin, I haven't changed the content of source page, will be updating that after Kubernetes plugin extraction.

TBH, I don't see a reason why not to put the Kubernetes related stuff directly under `sources/kubernetes.md`. I did sth like that for `kubectl` already. So in the `index.md` you have only information that are valid in general for all sources, rest is in dedicated docs.

Reply:
I haven't put plugin related things in index.md without kubernetes extraction as plugin. If you think it would be good, I will add them no worries.